src/bin/zmcontrol

```diff
@@ -377,6 +368,7 @@ sub doShutdown {
     my $rrc = 0;
     foreach ( sort { $stoporder{$b} <=> $stoporder{$a} } keys %allservices ) {
         Zimbra::Mon::Logger::Log( "info", "Stopping $_ via zmcontrol" );
+        if ( $_ eq "zimlet" || $_ eq "zimbraAdmin" || $_ eq "zimbra" ) { next; }
```

Comment:
Careful because if these services are up we won't stop them. Same for vice-versa. For the status is ok to skip to next iteration. We just have to test what happens.

Reply:
I checked the services are stopping and starting as expected. But please re-check to confirm. Is it related to https://github.com/Zextras/carbonio-core-utils/pull/29#discussion_r1090628787 ?

Reply:
> It would be nice to know for future improvements!

Yes, I checked and they all are managed by `/opt/zextras/bin/zmstorectl` the binding for each service to command to execute to perform any action is set https://github.com/Zextras/carbonio-core-utils/blob/4.0.15/src/bin/zmcontrol#L81 
- the **start** happens here https://github.com/Zextras/carbonio-core-utils/blob/4.0.15/src/bin/zmcontrol#L356
- the **stop** happens here https://github.com/Zextras/carbonio-core-utils/blob/4.0.15/src/bin/zmcontrol#L404

Reply:
So are we going to keep or remove that `if` condition?

Reply:
Cheking the logic of `/opt/zextras/bin/zmstorectl` it seems internally it just start,stop these two things: 
- mysql.server
- zmmailboxdctl   <-  this will start, stop and do status check on jetty.

Why our services are starting and stopping properly is because one of these mailbox, service or both are triggering the start, stop for us. so even if we skipped zimlet zimbraAdmin zimbra jetty start stop will be triggered by mailbox, service.

So, keep it as we are gucci :100:

Reply:
I think we can leave it since starting/stopping "service webapp", managed by zmstorectl, will also stop the others